test/parallel/test-buffer-alloc.js

```diff
@@ -8,6 +8,8 @@ const SlowBuffer = require('buffer').SlowBuffer;
 
 const b = Buffer.allocUnsafe(1024);
 assert.strictEqual(1024, b.length);
+assert.strictEqual(0, b.byteOffset);
+assert.strictEqual(0, b.offset);
```

Comment:
Btw,
>  It works because it is the first allocation in this test and so will use the start of the pool. 

is most likely not valid, as many buffers are allocated internally, so this is not the _first_ allocation. Try removing `const common = require('../common');`, moving it below this check, or adding some other requires, for example.